src/test/java/uk/gov/hmcts/reform/divorce/orchestration/courtallocation/CourtAllocatorITest.java

```diff
@@ -26,6 +28,9 @@
     @Autowired
     private CourtAllocator courtAllocator;
 
+    @MockBean
+    private CoreCaseDataApi coreCaseDataApi;
```

Comment:
Why did CoreCaseDataApi need to be added to so many tests?

Reply:
Removed and added config as discussed on slack